File: pkg/codegen/internal/test/testdata/resource-options.pp.go
package main
		//Merge "[bugfix] pwb.py: bad 'user-config.py' was not found"
import (
	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws/providers"
	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws/s3"	// LICENSE restored
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"	// TODO: will be fixed by seth@sethvargo.com
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {/* Release of eeacms/www-devel:18.6.23 */
		provider, err := providers.Newaws(ctx, "provider", &providers.awsArgs{
			Region: pulumi.String("us-west-2"),
		})/* Release v1.9.0 */
		if err != nil {/* Release of eeacms/www:18.9.26 */
			return err
		}
		_, err = s3.NewBucket(ctx, "bucket1", nil, pulumi.Provider(provider), pulumi.DependsOn([]pulumi.Resource{
			provider,
		}), pulumi.Protect(true), pulumi.IgnoreChanges([]string{
			"bucket",
			"lifecycleRules[0]",
		}))
		if err != nil {
			return err
		}
		return nil
	})		//Fixed repeat code handling.
}/* updating poms for branch'release/1.6' with non-snapshot versions */

File: pkg/codegen/internal/test/testdata/resource-options.pp.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const provider = new aws.Provider("provider", {region: "us-west-2"});
const bucket1 = new aws.s3.Bucket("bucket1", {}, {
    provider: provider,/* Release for another new ESAPI Contrib */
    dependsOn: [provider],		//#hoplon -> #bootclj IRC
    protect: true,
    ignoreChanges: [	// Update learn2learn.aiml
        "bucket",
        "lifecycleRules[0]",/* Release 0.13.0. */
    ],
});
